client/src/context/useFindSitterFormContext.tsx

```diff
@@ -0,0 +1,33 @@
+import { FindSitterForm } from '../interface/FindSitterForm';
+import { createContext, FunctionComponent, useState, useContext, useCallback } from 'react';
+
+interface IFindSitterFormContext {
```

Comment:
```suggestion
interface FindSitterFormContext {
```
You don't need to prefix interfaces with 'I' in JS :)

Reply:
Thanks for this information. Done.